staging/src/k8s.io/dynamic-resource-allocation/kubeletplugin/draplugin.go

```diff
@@ -272,12 +272,12 @@ type draPlugin struct {
 // If the plugin will be used to publish resources, [KubeClient] and [NodeName]
 // options are mandatory.
 //
-// By default, the DRA driver gets registered so that the plugin is compatible
-// with Kubernetes >= 1.32. To be compatible with Kubernetes >= 1.31, a driver
-// has to ask specifically to register only the alpha gRPC API, i.e. use:
-//
-//	Start(..., NodeV1beta1(false))
-func Start(ctx context.Context, nodeServer interface{}, opts ...Option) (result DRAPlugin, finalErr error) {
+// The DRA driver decides which gRPC interfaces it implements. At least one
+// implementation of [drapbv1alpha4.NodeServer] or [drapbv1beta1.DRAPluginServer]
+// is required. Implementing drapbv1beta1.DRAPluginServer is recommended for
+// DRA driver targeting Kubernetes >= 1.32. To be compatible with Kubernetes 1.31,
+// DRA drivers must implement only [drapbv1alpha4.NodeServer].
```

Comment:
Is this last sentence correct? it seems to contradict the previous if "DRA drivers must implement only" the alpha version

Reply:
The last sentence *is* "DRA drivers must implement only". Can you clarify which two sentences contradict each other?

Reply:
oh, my bad, missed the comma 😄

Reply:
Agreed, that is a bit subtle. Perhaps starting that sentence on a new line would have been more obvious, but I'll merge it now as-is.

/hold cancel